data/json/items/gun/762.json

```diff
@@ -236,5 +236,91 @@
+        "weight": 1
+      }
+    ],
+    "weight": "2900 g",
+    "volume": "2000 ml",
+    "longest_side": "84 cm",
+    "price": 200000,
+    "//": "Couldn’t find exact price information on original VZ assault rifles, so did some guestimation based off of prices for similar Russian automatic weapon’s's parts kits.",
+    "price_postapoc": 4000,
+    "to_hit": -1,
+    "bashing": 12,
+    "material": [ "steel", "plastic", "wood" ],
+    "color": "brown",
+    "ammo": [ "762" ],
+    "dispersion": 150,
+    "durability": 10,
+    "min_cycle_recoil": 1950,
+    "modes": [ [ "DEFAULT", "semi-auto", 1 ], [ "AUTO", "auto", 4 ] ],
+    "valid_mod_locations": [
+      [ "accessories", 4 ],
+      [ "barrel", 1 ],
+      [ "bore", 1 ],
+      [ "brass catcher", 1 ],
+      [ "grip", 1 ],
+      [ "mechanism", 4 ],
+      [ "muzzle", 1 ],
+      [ "rail mount", 1 ],
+      [ "sights mount", 1 ],
+      [ "sling", 1 ],
+      [ "stock", 1 ],
+      [ "stock accessory", 2 ],
+      [ "underbarrel mount", 1 ]
+    ],
+    "pocket_data": [ { "pocket_type": "MAGAZINE_WELL", "item_restriction": [ "vz58mag30", "vz58mag10" ] } ]
+  },
+  {
+    "id": "vz58_v",
+    "copy-from": "vz58_p",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "name": { "str": "VZ paratrooper carbine" },
+    "description": "A cold war era Czechoslovak assault rifle with a side-folding metal buttstock, likely constructed stateside from a pre-1986 imported parts kit or brought back from a 3rd world war zone.  Externally similar to the AK family of rifles, this weapon nevertheless only shares the former’s 7.62x39mm chambering.  Featuring no parts interchangeability with its Russian doppelganger, this firearm makes use of a completely different internal operating system, is far more svelte than its other Warsaw Pact counterparts, and feeds from proprietary box magazines.",
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "vz58_v",
+        "name": { "str": "Samopal VZ 58 V" },
+        "description": "This lightweight and relatively svelte cold war assault rifle was originally developed for use by the Czechoslovak army, in place of the various licenced Kalashnikov rifles which were then prevalent around eastern Europe.  Baring more than a passing resemblance to the AK-47, this weapon nevertheless features a completely different array of internals and possesses no parts interchangeability with its Russian counterpart.  Making use of proprietary magazines and a 15.4 inch barrel, the VZ. 58 is regarded by some to be the Kalashnikov’s lesser known superior.  Likely reconstructed from a pre-1986 imported parts kit, or brought back from a 3rd world war zone, this particular weapon features a removable side-folding buttstock and a 'V' suffixed designation, which both mark it as an original airborne or crew issued model.",
+        "weight": 1
+      }
+    ],
+    "built_in_mods": [ "folding_stock" ]
```

Comment:
I might just do this as a variant of the above one, and have all the spawns on this variant also spawn with a folding stock (like the m38dmr).

Reply:
Also the folding stocks on these are made to be removable to easily swap between the regular and paratrooper variants, so I don't think these should be ``built_in``:
https://youtu.be/iCqElIDUzeI?t=450